doc/releases/changelog-0.36.0.md

```diff
@@ -215,23 +215,46 @@
 
 <h4>Simulate mixed-state qutrit systems 3️⃣</h4>
 
-* Functions `measure_with_samples` and `sample_state` have been added to the new `qutrit_mixed` module found in
- `qml.devices`. These functions are used to sample device-compatible states, returning either the final measured state or value of an observable.
+* Mixed qutrit states can now be simulated with the `DefaultQutritMixed` 
+  device (shortname: `"default.qutrit.mixed"`).
+  [(#5495)](https://github.com/PennyLaneAI/pennylane/pull/5495)
+  [(#5451)](https://github.com/PennyLaneAI/pennylane/pull/5451)
+  [(#5186)](https://github.com/PennyLaneAI/pennylane/pull/5186)
   [(#5082)](https://github.com/PennyLaneAI/pennylane/pull/5082)
+  [(#5213)](https://github.com/PennyLaneAI/pennylane/pull/5213)
 
-* Fixed differentiability for Hamiltonian measurements in new `qutrit_mixed` module. 
-  [(#5186)](https://github.com/PennyLaneAI/pennylane/pull/5186)
+  Thanks to external contributors from the University of British Columbia, mixed 
+  qutrit circuits are now available to program, having access to the same 
+  operations, measurements, and functionality as circuits programmed on 
+  `DefaultQutrit`.
 
-* Added `simulate` function to the new `qutrit_mixed` module in `qml.devices`. This allows for simulation of a 
-  noisy qutrit circuit with measurement and sampling.
-  [(#5213)](https://github.com/PennyLaneAI/pennylane/pull/5213)
+  ```python
+  dev = qml.device("default.qutrit.mixed", wires=1)
 
-* Created the `DefaultQutritMixed` class, which inherits from `qml.devices.Device`, with an implementation 
-  for `preprocess`.
-  [(#5451)](https://github.com/PennyLaneAI/pennylane/pull/5451)
+  def circuit():
+      qml.TRY(0.1, wires=0)
 
-* Implemented `execute` on `qml.devices.DefaultQutritMixed` device, `execute` can be used to simulate noisy qutrit based circuits.
-  [(#5495)](https://github.com/PennyLaneAI/pennylane/pull/5495)
+  @qml.qnode(dev)
+  def shots_circuit():
+      circuit()
+      return qml.sample(), qml.expval(qml.GellMann(wires=0, index=1))
+
+  @qml.qnode(dev)
+  def density_matrix_circuit():
+      circuit()
+      return qml.state()
+  ```
+
+  ```pycon
+  >>> shots_circuit(shots=5)
+  (array([0, 0, 0, 0, 0]), 0.19999999999999996)
+  >>> density_matrix_circuit()
+  tensor([[0.99750208+0.j, 0.04991671+0.j, 0.        +0.j],
+        [0.04991671+0.j, 0.00249792+0.j, 0.        +0.j],
+        [0.        +0.j, 0.        +0.j, 0.        +0.j]], requires_grad=True)
+  ```
+
+  Keep your eyes peeled for qutrit noise model support in the coming releases!
```

Comment:
```suggestion
  However, there's one crucial ingredient that we still need to add: support for qutrit noise
  operations. Keep your eyes peeled for this to arrive in the coming releases!
```
Just want to address that we know it's a big piece that's still missing, but that we're on it.